cqf-fhir-cr/src/main/java/org/opencds/cqf/fhir/cr/measure/r4/R4MultiMeasureService.java

```diff
@@ -57,9 +59,15 @@ public R4MultiMeasureService(
         subjectProvider = new R4RepositorySubjectProvider(measureEvaluationOptions.getSubjectProviderOptions());
 
         r4Processor = new R4MeasureProcessor(
-                repository, this.measureEvaluationOptions, subjectProvider, r4MeasureServiceUtils);
+                repository,
+                this.measureEvaluationOptions,
+                subjectProvider,
+                r4MeasureServiceUtils,
+                populationBasisValidator);
```

Comment:
Question: do we need to pass in populationBasisValidator so soon? Wouldn't we only need to use this in MeasureEvaluator?

Reply:
I pushed down instantiating of both classes as far down the constructor hierarchy as I could.